third_party/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.runtimes</groupId>
```

Comment:
maven convention over configuration.
child modules reference their parent.

proposing an alternate approach then.
Drop `third_party/pom.xml` and just modify the parent `[jetty-runtime]/pom.xml` to add ...

``` xml
    <module>third_party/dandelion</module>
    <module>third_party/spring_petclinic</module>
```

With those specific poms not referencing the `[jetty-runtime]/pom.xml` parent pom.

We should make an effort to modify the `third_party/dandelion/pom.xml` and `third_party/spring_petclinic/pom.xml` to not deploy (snapshots or releases).
These modifications should appear as a separate PR to be very clear about the changes.

We don't need to for the testcases from Issue #33 anyway (we can rely on reactor and/or local repository just fine)